src/state/stake/hooks.ts

```diff
@@ -1083,88 +1095,83 @@ export function useDualStakingInfo(
   );
 }
 
-export function useLairInfo(): LairInfo {
-  const { account } = useActiveWeb3React();
-
-  let accountArg = useMemo(() => [account ?? undefined], [account]);
-
-  const inputs = ['1000000000000000000'];
-
-  const lair = useLairContract();
-  const quick = useQUICKContract();
-
-  const _dQuickTotalSupply = useSingleCallResult(lair, 'totalSupply', []);
-
-  const quickBalance = useSingleCallResult(lair, 'QUICKBalance', accountArg);
-  const dQuickBalance = useSingleCallResult(lair, 'balanceOf', accountArg);
-  const dQuickToQuick = useSingleCallResult(lair, 'dQUICKForQUICK', inputs);
-  const quickToDQuick = useSingleCallResult(lair, 'QUICKForDQUICK', inputs);
-
-  accountArg = [GlobalConst.addresses.LAIR_ADDRESS ?? undefined];
-
-  const lairsQuickBalance = useSingleCallResult(quick, 'balanceOf', accountArg);
-
-  useEffect(() => {
-    getOneDayVolume();
-  }, []);
+export function useOldLairInfo(): LairInfo {
+  const lairContract = useLairContract();
+  const quickContract = useQUICKContract();
+  const lairAddress = GlobalConst.addresses.LAIR_ADDRESS;
+  const quickToken = GlobalValue.tokens.COMMON.OLD_QUICK;
+  const dQuickToken = GlobalValue.tokens.COMMON.OLD_DQUICK;
 
-  return useMemo(() => {
-    return {
-      lairAddress: GlobalConst.addresses.LAIR_ADDRESS,
-      dQUICKtoQUICK: new TokenAmount(
-        returnTokenFromKey('QUICK'),
-        JSBI.BigInt(dQuickToQuick?.result?.[0] ?? 0),
-      ),
-      QUICKtodQUICK: new TokenAmount(
-        returnTokenFromKey('DQUICK'),
-        JSBI.BigInt(quickToDQuick?.result?.[0] ?? 0),
-      ),
-      dQUICKBalance: new TokenAmount(
-        returnTokenFromKey('DQUICK'),
-        JSBI.BigInt(dQuickBalance?.result?.[0] ?? 0),
-      ),
-      QUICKBalance: new TokenAmount(
-        returnTokenFromKey('QUICK'),
-        JSBI.BigInt(quickBalance?.result?.[0] ?? 0),
-      ),
-      totalQuickBalance: new TokenAmount(
-        returnTokenFromKey('QUICK'),
-        JSBI.BigInt(lairsQuickBalance?.result?.[0] ?? 0),
-      ),
-      dQuickTotalSupply: new TokenAmount(
-        returnTokenFromKey('DQUICK'),
-        JSBI.BigInt(_dQuickTotalSupply?.result?.[0] ?? 0),
-      ),
-      oneDayVol: oneDayVol,
-    };
-  }, [
-    quickBalance,
-    dQuickBalance,
-    _dQuickTotalSupply,
-    lairsQuickBalance,
-    dQuickToQuick,
-    quickToDQuick,
-  ]);
+  return useLairInfo(
+    lairContract,
+    quickContract,
+    lairAddress,
+    quickToken,
+    dQuickToken,
+  );
 }
 
 export function useNewLairInfo(): LairInfo {
```

Comment:
Lets keep this function but as I said in the previous comments that we can't showcase new lair in the UI as yet